scripts/upsify-daq-pkgs.py

```diff
@@ -240,10 +279,10 @@ def create_ups_pkg(install_dir, source_dir, equal, dqual, dest_dir, pkg_name="")
                         help="flag for the 'debug' qualifer ('prof' if unset).")
 
     args = parser.parse_args()
-    workdir = args.work_dir
+    workdir = path.abspath(args.work_dir)
```

Comment:
Other places used `os.path`, e.g. `os.path.exist` or `os.path.join` above.

So I recommend change this to `os.path.abspath`, and remove `import os.path as path` at the top of the file.